Automattic/amp-wp
# Sanitize PDF embeds

## Summary


Fixes #6083.

(Note that being unable to see the embed on the AMP page is expected if the PDF is hosted locally.)

## Todo

- [x] Determine why the `amp-google-document-embed` component is not being rendered in the Twenty Twenty theme.

## Demo

![image](https://user-images.githubusercontent.com/16200219/115943715-33c26580-a4a1-11eb-8b8d-ba9643a2a598.png)

## QA Instructions

1. Create a new post
2. Insert a file block into the editor
3. Upload a PDF document
4. Verify that the PDF document is displayed on the AMP page

## Checklist
